stable/mlrun-kit/values.yaml

```diff
@@ -137,10 +139,10 @@ mlrun:
       annotations: ~
 
   httpDB:
-    dbType: "sqlite"
-    dirPath: "/mlrun/db"
-    dsn: "sqlite:////mlrun/db/mlrun.db?check_same_thread=false"
-    oldDsn: ""
+    dbType: mysql
+    dsn: mysql+pymysql://root@mlrun-db:3306/mlrun
+    oldDsn: sqlite:////mlrun/db/mlrun.db?check_same_thread=false
+
```

Comment:
Don't do this change yet. There's an issue with the CI that's failing to bring up mysql like this